datalad/customremotes/archives.py

```diff
@@ -393,8 +394,16 @@ def _transfer(self, cmd, key, path):
                 # so
                 pwd = getpwd()
                 lgr.debug(u"Getting file {afile} from {akey_path} while PWD={pwd}".format(**locals()))
+                was_extracted = self.cache[akey_path].is_extracted
                 apath = self.cache[akey_path].get_extracted_file(afile)
                 link_file_load(apath, path)
+                if not was_extracted and self.cache[akey_path].is_extracted:
+                    self.info("%s special remote is using an extraction cache "
+                              "under %s. You may want to call 'datalad clean' "
+                              "to remove it afterwards." %
```

Comment:
```suggestion
                    self.info("%s special remote created an extraction cache "
                              "under %s. Remove it with DataLad's 'clean' command "
                              "to save disk space." %
```

There can be performance loss when it is removed. I would prefer concrete advice when to do what, rather than a more nebulous statement.

Reply:
Yeah, I see that. However, this is the special remote itself. Technically it's kinda wrong to point to datalad commands to begin with + long message would be there all the time. Wondering: We usually call annex with `--json` by now, giving us an info object that we decide to spit out (rather than annex itself). So, we could use that for a short message that we catch in datalad and can potentially decide to enhance or swallow from within a context where we know much more about the situation than within the special remote. Not too clear to me at this point, just thinking about principles how to deal with messaging better in general.